packages/ember-glimmer/lib/utils/references.js

```diff
@@ -53,6 +53,25 @@ export class UpdatableReference extends RootReference {
   }
 }
 
+export class GetHelperReference {
+  constructor(sourceReference, pathReference) {
+    this.sourceReference = sourceReference;
+    this.pathReference = pathReference;
+  }
+
+  isDirty() { return true; }
+
+  value() {
+    return this.sourceReference.get(this.pathReference.value()).value();
```

Comment:
Just chatted with @chancancode. I don't think we can use `Ember.get`, because `Ember.get` is not **always** 1:1 with `PathReference#get`. The most notable example of this today is unbound references, which cache a constant reference.

For now, going through the proper interface seems fine.